frontend/src/features/VesselSidebar/Controls/ControlsSummary.tsx

```diff
@@ -3,7 +3,9 @@ import styled from 'styled-components'
 
 import { InfractionsSummary } from './InfractionsSummary'
 import { LastControl } from './LastControl'
+import { LawReminders } from './LawReminders'
 import { theme } from '../../../ui/theme'
+import { ReactComponent as CautionSVG } from '../../icons/Attention_controles.svg'
```

Comment:
Pourrais-tu rajouter un `TODO Add the icon to https://github.com/MTES-MCT/monitor-ui` ?
Je viens de voir que cette icône est manquante dans notre bibliothèque des icônes, on la rajoutera dans un deuxième temps!